client/reader/following-manage/subscriptions.jsx

```diff
@@ -124,7 +120,7 @@ class FollowingManageSubscriptions extends Component {
 							sites={ sortedFollows }
 							width={ width }
 							remoteTotalCount={ sortedFollows.length }
-							forceRefresh={ [ feeds, sortedFollows ] }
+							forceRefresh={ sortedFollows }
```

Comment:
i dropped forceRefresh and the only bug i was running into was when clicking "Show More" the placeholders would stay until you start scrolling